packages/calcite-components/src/components/accordion/accordion.scss

```diff
@@ -2,13 +2,8 @@
  * CSS Custom Properties
  *
  * @prop --calcite-accordion-space: defines space around and between accordion item elements. Other spacing is calculated at a +4.
- * @prop --calcite-accordion-border-color: The border around an accordion.
+ * @prop --calcite-accordion-border-color: The border around an accordion and accordion items.
  * @prop --calcite-accordion-background: The background color of the accordion and accordion items.
```

Comment:
For consistency, should we deprecate this `background` and add a `background-color` instead? Similar to https://github.com/Esri/calcite-design-system/pull/8662/files

Or, maybe just deprecate without replacement if this background color never "gets seen" because only the Accordion Item has visible background?

Reply:
missed this one. Thanks!